README.md

```diff
@@ -92,13 +95,45 @@ For example, a m4.4xlarge node can have up to 8 ENIs, and each ENI can have up t
 [Elastic Network Interfaces documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html) for details.
 
 For a detailed explanation, see [`WARM_ENI_TARGET`, `WARM_IP_TARGET` and `MINIMUM_IP_TARGET`](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/docs/eni-and-ip-target.md).
-
-
 ## Privileged mode
 
 VPC CNI makes use of privileged mode (`privileged: true`) in the manifest only for its init container. This elevated privilege is required to set the networking kernel parameters.
 
-### CNI Configuration Variables<a name="cni-env-vars"></a>
+## Network Policies
+
+VPC CNI versions <<NP_RELEASE_VERSION>> and greater support [Kubernetes Network Policies.](https://kubernetes.io/docs/concepts/services-networking/network-policies/). Network Policies specify how pods can communicate over the network, at the IP address or port level. The VPC CNI implements the Kubernetes [NetworkPolicy](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#networkpolicy-v1-networking-k8s-io) API. Network policies generally include a pod selector, and Ingress/Egress rules. 
+
+For EKS clusters, review [Configure your cluster for Kubernetes network policies](<<NP_AWS_DOCS_LINK>>) in the Amazon EKS User Guide.
+
+The AWS VPC CNI implementation of network policies may be enabled in self-managed clusters. This requires both the VPC CNI agent, and the [Amazon Network Policy Controller for Kubernetes](https://github.com/aws/amazon-network-policy-controller-k8s). <<is name final?>>. 
+
+default allow where all pod-to-pod communication is allowed unless NetworkPolicy is implemented to restrict explicitly
```

Comment:
nit: Something like "In Kubernetes, by default, all pod-to-pod communication is allowed. Communication can be restricted with NetworkPolicy objects"